src/music/Instrument.ts

```diff
@@ -0,0 +1,13 @@
+import { Note } from "./Notes.js";
+
+export default abstract class Instrument {
+
+    volume: number;
+
+    constructor(volume: number) {
+        this.volume = volume;
+    }
```

Comment:
`constructor(public volume: number) {}` is more consistent with the rest of the codebase